src/components/instant-apps-interactive-legend/support/helpers.ts

```diff
@@ -230,7 +230,7 @@ function generateQueryExpression(info: any, field: string, infoIndex: number, le
         // Types unique symbols - 'Other' category
         const expressionList = [] as string[];
         legendElementInfos?.forEach(legendElementInfo => {
-          if (info.value) {
+          if (legendElementInfo?.value) {
```

Comment:
Can the value be 0?

Reply:
No, legendElementInfo values are optional objects with data about the info:
https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Legend-support-ActiveLayerInfo.html#SymbolTableElementInfo

If it doesn't exist, the generated expression will fall back to an empty string, as an expression won't be needed since a value doesn't exist